scripts/safety.py

```diff
@@ -252,8 +257,13 @@ def spin(self):
             # Step 2: check if robot should kill
             if self._kill_flag:
                 self.kill()
-
-            self._spin_rate_control.sleep()
+                rs.disable()
```

Comment:
The robot shouldn't enable itself again if it fails. Imagine you have a code that kept sending it bad commands. You first noticed it and disabled Baxter, the malicious code will still be running and kept sending commands. If you re-enabled Baxter, there is no guarantee that the malicious code will be disabled. It's safer if the baxter is disabled until a programmer/admin re-enables it.